contracts/Aavegotchi/facets/WhitelistFacet.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.1;
+
+import {LibMeta} from "../../shared/libraries/LibMeta.sol";
+import {Modifiers, WHITELIST_LIMIT} from "../libraries/LibAppStorage.sol";
+
+contract WhitelistFacet is Modifiers {
+    event WhitelistCreated(uint256 indexed whitelistId, address indexed owner, address[] addresses);
+
+    event WhitelistUpdated(uint256 indexed whitelistId, address indexed owner, address[] addresses);
+
+    function createWhitelist(address[] memory _whitelistAddresses) external {
```

Comment:
It is more gas efficient to mark arrays as `calldata` than memory.  I suggest changing this to this:
`function createWhitelist(address[] calldata _whitelistAddresses)`

Reply:
OK.